tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -698,6 +709,38 @@ void TangoRosNode::StopPublishing() {
   }
 }
 
+bool TangoRosNode::SaveMap(std::string map_name) {
+  TangoErrorType result;
+  TangoUUID map_uuid;
+  result = TangoService_saveAreaDescription(&map_uuid);
+  if(result != TANGO_SUCCESS) {
+    LOG(ERROR) << "Error while saving area description, error: " << result;
+    return false;
+  }
+  TangoAreaDescriptionMetadata metadata;
+  result = TangoService_getAreaDescriptionMetadata(map_uuid, &metadata);
+  if(result != TANGO_SUCCESS) {
+    LOG(ERROR) << "Error while trying to access area description metada, error: " << result;
```

Comment:
@jubeira I guess you suffered from a typo as well :wink: , it should be `metadata` :smile: